docs/api/insert.md

```diff
@@ -0,0 +1,293 @@
+
+### Input
+
+- the scheme of `payload.v1.Insert.Request`
+
+  ```bash
+  message Insert {
+      message Request {
+          Object.Vector vector = 1 [ (validate.rules).repeated.min_items = 2 ];
+          Config config = 2;
+      }
+
+      message Config {
+          bool skip_strict_exist_check = 1;
+          Filter.Config filters = 2;
+          int64 timestamp = 3;
+      }
+  }
+
+  message Object {
+      message Vector {
+          string id = 1 [ (validate.rules).string.min_len = 1 ];
+          repeated float vector = 2 [ (validate.rules).repeated.min_items = 2 ];
+      }
+  }
+  ```
+
+  - Insert.Request
+    |field|type|label|required|desc.|
+    |:---:|:---|:---|:---:|:---|
+    |vector|Object.Vector| | \* | the information of vector |
+    |config|Config| | \* | the configuration of the insert request |
+
+  - Insert.Config
+    |field|type|label|required|desc.|
+    |:---:|:---|:---|:---:|:---|
+    |skip_strict_exist_check|bool| | | check the same vector is already inserted or not.<br>the same ID is not indexed if the value is `true`|
+    |timestamp|int64| | | the timestamp of the vector inserted.<br>if it is N/A, the current time will be used.
+    |filters|Filter.Config| | | configuration for filter |
+
+  - Object.Vector
+    |field|type|label|required|desc.|
+    |:---:|:---|:---|:---:|:---|
+    |id|string| | \* | the ID of a vector. ID consists 1 or more strings. |
+    |vector|float| repeated(Array[float]) | \* | the vector data. its dimension is between 2 and 65,536.|
+
+### Output
+
+- the scheme of `payload.v1.Object.Location`
+
+  ```bash
+  message Object {
+      message Location {
+        string name = 1;
+        string uuid = 2;
+        repeated string ips = 3;
+      }
+  }
+  ```
+
+  - Object.Location
+    |field|type|label|desc.|
+    |:---:|:---|:---|:---|
+    |name|string| | the name of vald agent pod which has been inserted the request vector. |
+    |uuid|string| | the ID of the inserted vector. It is the same as Object.Vector |
+    |ips|string| repeated(Array[string]) | the ip list of `vald-agent` pods which has been inserted the request vector. |
+
+### Status Code
+
+| code | desc.            |
+| :--: | :--------------- |
+|  0   | OK               |
+|  3   | INVALID_ARGUMENT |
+|  6   | ALREADY_EXISTS   |
+|  13  | INTERNAL         |
+
+## StreamInsert RPC<sup>recommended</sup>
+
+StreamInset RPC is the method  to add new multiple vectors by the [bidirectional streaming RPC](https://grpc.io/docs/what-is-grpc/core-concepts/#bidirectional-streaming-rpc).<br>
+It can communicate between client and server in any order.
+Each Insert request and response are independent.
+It's a recommended method to insert the large amount of vectors.
+
+### Input
+
+- the scheme of `payload.v1.Insert.Request stream`
+
+  ```bash
+  message Insert {
+      message Request {
+          Object.Vector vector = 1 [ (validate.rules).repeated.min_items = 2 ];
+          Config config =2;
```

Comment:
Sorry, I cannot find same point.:bow:
Please fix the same way.:pray:

Reply:
```suggestion
          Config config = 2;
```